content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-contract.md

```diff
@@ -66,15 +65,45 @@ class MarsVoucherContract : Contract {
 }
 ```
 
-### Add commands
+### Connect the `MarsVoucherContract` to the `MarsVoucher` state
+
+After creating the contract class in a CorDapp, you must connect the contract to its correlating state. Add the `@BelongsToContract` annotation *in the state class* to establish the relationship between a state and a contract. Without this, your state does not hold a relationship to the contract that is used to verify it.
 
-Commands indicate the transaction's intent — what type of actions performed by the state the contract can verify.
+1. Open your `MarsVoucher` class.
+2. Insert the `@BelongsToContract` annotation with the `MarsVoucherContract` just before the definition of the `MarsVoucher` data class.
```

Comment:
it would be better to just say "annotate the class with thw blah annotation", we're asuming here people know Jaba abd what an annotation is